.github/workflows/publish-pypi-deploy.yml

```diff
@@ -12,52 +12,32 @@ jobs:
       fail-fast: true
       matrix:
         python-version: [ '3.10' ]
-
+    environment:
+      name: testpypi  # pypi
+      url: https://test.pypi.org/legacy/  # https://pypi.org/p/pydicom
```

Comment:
I guess this is for testing only. Won't you need to set a password for testpypi?

Reply:
It's not needed now with this trusted publisher concept. I set up this workflow as trusted in both pypi and test pypi.

Reply:
Ah ok, thanks!